public/views/play-with.html

```diff
@@ -0,0 +1,53 @@
+<nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
+  <div class="container custom-container">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header page-scroll">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+      <span class="sr-only">Toggle navigation</span>
+      Menu
+      <i class="fa fa-bars"></i>
+      </button>
+      <a class="navbar-brand-custom" href="#">
+      <img alt="Brand" src="./img/cards-for-humanity.png" class="img-responsive" width="70">
```

Comment:
Say NO to inline styling.  Move the width styling to a css file. 

Also, since you are applying a class `img-responsive` to the image, you don't want to set a static `width: 70px`. You should make this `max-width` instead.